llvm_ir_dataset_utils/util/bitcode_module.py

```diff
@@ -232,6 +232,22 @@ def get_instruction_counts(bitcode_module, additional_passes=''):
     ]
 
 
+def get_fields_total_counts(bitcode_module, additional_passes='', names=False):
+  properties_or_error = get_function_properties_module(bitcode_module,
+                                                       additional_passes)
+  if properties_or_error[0]:
+    return None
+  else:
+    if (names):
+      return [key[0] for key in properties_or_error[1].items()]
+    else:
+      return [
+          str(sum([int(i)
```

Comment:
Why are you converting the integers back to a string? They should probably stay as integers once you parse them.